### PR TITLE
feat: Added verified logo to README

### PR DESCRIPTION
**Pull request by bitsandfoxes**
#skip-changelog

### CODE REVIEW

File: package/README.md [hunk 1]
```diff
@@ -1,9 +1,13 @@
 <p align="center">
-  <a href="https://sentry.io" target="_blank" align="center">
-    <img src="https://sentry-brand.storage.googleapis.com/sentry-logo-black.png" width="280">
+  <a href="https://sentry.io" target="_blank" align="left">
+    <img src="https://raw.githubusercontent.com/getsentry/sentry-unity/main/.github/sentry-wordmark-dark-400x119.svg" width="280">
+  </a>
+  <a href="https://docs.sentry.io/platforms/unity/" target="_blank" align="right">
+    <img src="https://raw.githubusercontent.com/getsentry/sentry-unity/main/.github/unity-verified-logo.svg" width="280">
```
Comment (vaind) [inline package/README.md:6]: maybe use arelative image path?
```suggestion
    <img src="./.github/sentry-wordmark-dark-400x119.svg" width="280">
  </a>
  <a href="https://docs.sentry.io/platforms/unity/" target="_blank" align="right">
    <img src="./.github/unity-verified-logo.svg" width="280">
```

File: package/README.md [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
   </a>
   <br />
 </p>
+<p align="center">
```
Comment (vaind) [inline package/README.md:10]: why this?

File: package/README.md (hunk 1 above)
Comment (bitsandfoxes) [reply package/README.md:6]: This readme ends up on the package-repo https://github.com/getsentry/unity and relative paths won't work there.

File: package/README.md (hunk 2 above)
Comment (bitsandfoxes) [reply package/README.md]: good point. my bad

**Comment by lucas-zimerman** [comment]
I don't think it's required to do all the ci build only for a change on the readme file.

**Review by lucas-zimerman — APPROVED** [review]

**Pull request merged**